flink-libraries/flink-table/src/main/scala/org/apache/flink/table/codegen/AggregationCodeGenerator.scala

```diff
@@ -201,14 +242,22 @@ class AggregationCodeGenerator(
         j"""
            |  public final void accumulate(
            |    org.apache.flink.types.Row accs,
-           |    org.apache.flink.types.Row input)""".stripMargin
+           |    org.apache.flink.types.Row input) throws Exception """.stripMargin
 
       val accumulate: String = {
-        for (i <- aggs.indices) yield
+        for (i <- aggs.indices) yield {
+          val setDataView = if (accConfig.isDefined && accConfig.get.isUseState) {
+            genDataViewFieldSetter(s"acc$i", accConfig.get.accSpecs(i))
```

Comment:
Same as for `setAggregationResults()`. I think we should reuse the state objects.